pom.xml

```diff
@@ -129,6 +128,14 @@
         <micrometer.version>1.3.1</micrometer.version>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>test-container</id>
+            <name>Strimzi Kafka Container repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        </repository>
+    </repositories>
+
```

Comment:
Why do we need to add this? This seems to introduce circular dependency -> if you use the test containers here, but building a new version of test-containers depends on the container images build by this repo. If we need to have this kind of dependency I'm not sure we can really move the test container to separate repo.